### PR TITLE
Removed WebDriverManager Dependency from Teswiz framework

### PR DESCRIPTION
**Pull request by ngourkar**
Description - Teswiz framework already supports the latest selenium version 4.10.0 which has the capabilities of selenium manager. With this latest selenium version we don't need to explicitly setup the webdrivermanager. The selenium itself identifies the latest and supporting driver and runs the script on it.

Enhancement - 
1. Remove the dependency on Webdrivermanager (boni gracia's third party library)
2. Reduced the additional piece of code.

Observations:
Have tried and tested the scripts by running it on local system, docker and cloudplatform i.e. Browserstack.

All the observations, POC for this task and required documentation is present in this link.
[Documentation]([https://rilcloud-my.sharepoint.com/%3Aw%3A/g/personal/darshan3_s_ril_com/EeEbFrfFSWROn0m4oaM2W0MB8OVE1Py7ctslAfNDwS_4NA?e=WuRQjS](https://rilcloud-my.sharepoint.com/%3Aw%3A/g/personal/darshan3_s_ril_com/EeEbFrfFSWROn0m4oaM2W0MB8OVE1Py7ctslAfNDwS_4NA?e=WuRQjS&xsdata=MDV8MDF8fGY2YzJmNDdmYzBlZjQ4MDFlZWE3MDhkYjg2YzNhZmQzfGZlMWQ5NWE5NGNlMTQxYTU4ZWFiNmRkNDNhYTI2ZDlmfDB8MHw2MzgyNTE5NDcyMjk3MDM4NDd8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPakk1TnpsaU1XVmhMVGN3WTJRdE5EZzNPQzFpTmprMExXRTVOekJtTVRkbE5tWXlORjltTXpSbU1EWTFPUzA1WkdWbUxUUmlOREl0WWpJNE9DMHdPR0U1TVdKak9Ea3pPRGRBZFc1eExtZGliQzV6Y0dGalpYTXZiV1Z6YzJGblpYTXZNVFk0T1RVNU56a3lNakF6TXc9PXxhZmVlNDZhYzViZjc0YjU4ZWVhNzA4ZGI4NmMzYWZkM3xjMjFlOGYzMTU2NDU0ZmFmOGYxZmM4ZGZmMDQ4YjI3MQ%3D%3D&sdata=K2hjeS9HWVc5Uy9ZUVBvYmlodGRZOHlTRkg3dEdIL3UvVXM0ZVJsL2Nuaz0%3D&ovuser=fe1d95a9-4ce1-41a5-8eab-6dd43aa26d9f%2CNayan.Gourkar%40ril.com))

### CODE REVIEW

**Comment by anandbagmar** [comment]
@ngourkar - raise PR against removeWDM branch

File: src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java [hunk 1]
```diff
@@ -195,42 +191,6 @@ private static WebDriver createNewWebDriver(String forUserPersona, String browse
...
-            String[] getBrowserVersionFromDockerCommand = new String[] {"cURL -s --request GET " +
-                                                        "'http://localhost:" + Runner.getRemoteDriverGridPort() + "/status'" +
-                                                       " | jq -r -M  '.value" +
-                               ".nodes[].slots[].stereotype | select(.browserName == \"" + browserType + "\") " +
-                                                                "| .browserVersion '"};
-            CommandLineResponse commandLineResponse = CommandLineExecutor.execCommand(getBrowserVersionFromDockerCommand);
-            browserVersion = commandLineResponse.getStdOut().split("\n")[0];
-            LOGGER.info(String.format("%s browser version in docker container: %s", browserType, browserVersion));
-        }
-        WebDriverManager webDriverManager = WebDriverManager.getInstance(driverManagerType)
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java:220]: How is proxy handled in SeleniumManager?

File: src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java [hunk 2]
```diff
@@ -195,42 +191,6 @@ private static WebDriver createNewWebDriver(String forUserPersona, String browse
...
-                                                            .proxy(webDriverManagerProxyUrl);
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java:221]: if Proxy is not required, we need to remove it from the config.properties as well - WEBDRIVER_MANAGER_PROXY_KEY, right?

Comment (ngourkar) [reply src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java:221]: Proxy is not required and thus all the unused methods and properties are removed now.

**Pull request merged**